Gemfile

```diff
@@ -8,6 +8,7 @@ group :development, :unit_tests do
   gem 'simplecov', :require => false
   gem 'puppet_facts', :require => false
   gem 'json', :require => false
+  gem 'metadata-json-lint', :require => false
```

Comment:
This doesn't seem to be related to this PR, please make a separate one for this.